.github/workflows/verify.yaml

```diff
@@ -84,22 +84,14 @@ jobs:
     env:
       JACOCO: true
 
-    services:
-      postgres:
-        image: postgres:14.2
-        ports:
-          - 5432:5432
-        env:
-          POSTGRES_PASSWORD: password
-
     steps:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/gradle-setup
 
       - name: Postgresql Tests
         uses: ./.github/actions/run-tests
         with:
-          command: ./gradlew test -DincludeTags="PostgresqlIntegrationTest"
+          command: ./gradlew test -DincludeTags="ComponentTest"
```

Comment:
We either need to specify the directory (`-p extensions/store/sql test ...`), which would only execute tests in that folder, or we rename the job from `Postgresql-Integration-Tests` to just `Integration-Tests`, or even better, `Component-Tests`